build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -172,6 +172,30 @@ service ActionCache {
   }
 }
 
+// The repository cache API is used to download and cache external repositories.
```

Comment:
Meta comment, probably for @ola-rozenfeld : is it time to start splitting this API into multiple files? this one in particular feels like a reasonable candidate to be in its own file.

Reply:
I agree.  Once more it raises the question of separating out CAS as well, as it and the Digest, Directory, Tree types are probably the most reused in other contexts.